### PR TITLE
[Azure Sentinel Hackathon 2021] Azure Sentinel as new on premise SOAR replacement solution - IBM Qradar Connector

### PR DESCRIPTION
**Pull request by hieuttmmo**
When we were looking for hackathon ideas, we found that in common deployment scenario, Azure Sentinel just worked as a "complementary" or second SIEM, serving just for cloud environment and the primary SIEM solution is still typical on-premise product such as IBM QRadar or Splunk. Therefore, we want to do in opposite approach that **Sentinel will act as a primary SIEM/SOAR** - a single pane of glass for all security operation programs. Other SIEMs become the second, sending on premise signal data to Sentinel. Azure Sentinel is a new cyberdefense commander for cloud era. 

We think that this idea is a good fit for any enterprises in cloudification journey, when cloud becomes a major workload and the rest is on premise.

### CODE REVIEW

**Comment by NikTripathi** [comment]
@hieuttmmo  Please resolve conflicts.

File: Sample Data/Event/QradarOffenses.json [hunk 1]
```diff
@@ -0,0 +1,1222 @@
+[
```
Comment (NikTripathi) [inline Sample Data/Event/QradarOffenses.json:1]: Name of the sample data file should be same as legend name which is **QRadarOffense_CL**

**Review by NikTripathi — CHANGES_REQUESTED** [review]

**Comment by NikTripathi** [comment]
@aprakash13 Please review detections.

**Comment by NikTripathi** [comment]
@hieuttmmo please resolve conflicts. And please move this in solution folder.

File: Workbooks/WorkbooksMetadata.json [hunk 2]
```diff
@@ -1456,31 +1456,6 @@
       "subtitle": "",
       "provider": "Azure Sentinel Community" 
       },
-      {
```
Comment (v-jayakal) [inline Workbooks/WorkbooksMetadata.json:1459]: Why these workbook entries are removed?

**Review by v-jayakal — CHANGES_REQUESTED** [review]

File: Detections/Qradar/QRadarMultipleOffensesFromSingleIP.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+query: |
+  QRadarOffense_CL
+  | summarize count() by id_d, offense_source_s 
+  | where count_ > 5 and offense_source_s matches regex @"((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)"
+entityMappings:
+  - entityType: IP
+    fieldMappings:
+      - identifier: Address
+        columnName: offense_source_s
+version: 1.0.0
```
Comment (aprakash13) [inline Detections/Qradar/QRadarMultipleOffensesFromSingleIP.yaml:22]: Please add the "kind: Scheduled" property in the end.

**Review by aprakash13 — COMMENTED** [review]
There is just one detection that looks ok.  Have made a small suggestion to be fixed. 
Once the submitter fixes it as well as fixes the validation errors, please feel free to approve, if other components look ok.

**Comment by v-sabiraj** [comment]
Hey @hieuttmmo , please update the requested changes ,thanks.

**Comment by vmanojreddy** [comment]
Please resolve conflicts

**Comment by vmanojreddy** [comment]
@hieuttmmo Please move this in solution folder.

**Comment by hieuttmmo** [comment]
> @hieuttmmo Please move this in solution folder.

@vmanojreddy Just moved, sorry for the delay

**Comment by vmanojreddy** [comment]
@hieuttmmo  Workbook query's are failing please look into it. Thanks!!
image

**Comment by vmanojreddy** [comment]
@hieuttmmo please update the requested changes ,thanks!!!

**Comment by vmanojreddy** [comment]
@hieuttmmo please update the requested changes ,thanks!!!

**Comment by ashishsyal** [comment]
@hieuttmmo - Please let us know if you will be able to move this forward..

**Comment by hieuttmmo** [comment]
Can you guys check it again

**Comment by vmanojreddy** [comment]
@hieuttmmo Please resolve conflicts. thanks!!!

**Comment by hieuttmmo** [comment]
> @hieuttmmo Please resolve conflicts. thanks!!!

Conflict resolved

**Comment by hieuttmmo** [comment]
> @hieuttmmo Workbook query's are failing please look into it. Thanks!! image

I checked the table name again, it is correct. The might be the lack of the QradarOffenses_CL table in your Sentinel instance

**Comment by vmanojreddy** [comment]
@hieuttmmo Please move the files into solution folder. thanks!!!

**Comment by hieuttmmo** [comment]
> @hieuttmmo Please move the files into solution folder. thanks!!!

I moved the Detection rule and Workbook in to the Solutions/Qradar folder

**Comment by vmanojreddy** [comment]
@hieuttmmo Please update the azuredeploy short link https://aka.ms/sentinel-qradarconnector-azuredeploy. thanks!!!

**Comment by hieuttmmo** [comment]
> https://aka.ms/sentinel-qradarconnector-azuredeploy

Yes, I update the shorten link as per requested

**Review by aprakash13 — COMMENTED** [review]
There is just one query that looks ok from a query perspective.
Please feel free to merge if everything else looks good.

**Comment by vmanojreddy** [comment]
@hieuttmmo  Please resolve conflicts. thanks!!!

**Comment by v-dvedak** [comment]
@hieuttmmo, Please resolve conflicts.

**Comment by vmanojreddy** [comment]
@hieuttmmo We wanted to check on the status of PR [IBM Qradar Connector](https://github.com/Azure/Azure-Sentinel/pull/3158#)
#3158. PR is pending from long time. Let us know if any assistance is required for this PR. Per our standard operating procedures if no response is received in the next 7 business days we will close this PR. Thank you for your cooperation.

**Comment by vmanojreddy** [comment]
@hieuttmmo We wanted to check on the status of PR https://github.com/Azure/Azure-Sentinel/pull/3158#
https://github.com/Azure/Azure-Sentinel/pull/3158. PR is pending from long time. Let us know if any assistance is required for this PR. Per our standard operating procedures if no response is received in the next 7 business days we will close this PR. Thank you for your cooperation.

**Comment by v-spadarthi** [comment]
@hieuttmmo : Could you please resolve the conflicts

**Comment by v-spadarthi** [comment]
@hieuttmmo : Please resolve the conflicts. Thanks !

**Comment by v-dvedak** [comment]
Hey @hieuttmmo, we wanted to check on the status of this [PR](https://github.com/Azure/Azure-Sentinel/pull/3158)
. It is pending from more than 60 days. Let us know if any assistance is required for this PR. Per our standard operating procedures if no response is received in the next 7 business days we will close this PR. Thank you for your cooperation.

**Comment by v-spadarthi** [comment]
Hi @hieuttmmo, since we have not received response from last 7 days, we are closing your PR per our standard operating procedures. If you still need support for this issue, you can re-open the PR at any time.
If you do re-open, we simply request that you ensure the PR has response to the last request. Thank you for your cooperation.

**Pull request closed without merging**